### DISCUSSION TITLE
Wiki updates

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#209] [General]
One thing that's annoying with Github is that there are no notifications when someone updates the wiki pages. Therefore I thought I'd make a thread so people can alert when they've done it, and also discussion around what is there/should be there.

As a starter I just added the new scanner zoom variables to the [rules](https://github.com/swfans/swars/wiki/Game-Modifications-%E2%80%90-Rules.ini) page. PersuadedPersonWeaponsSellCostPermil is now also there, but I must admit I don't follow the exact algorithm of how that works @mefistotelis  could you see if it's a reasonable summary and/or give a better explanation of how it works?

**Comment by mefistotelis** [comment]
Looks good, though using the word "percentage" indicates incorrect unit.. will fix.

**Comment by geist22** [comment]
Yeah, I thought about opening such a thread too, since there is no way to leave comments when editing the wiki.

I've changed the explanation under **DailyScientistDeathChancePermil** on the Rules.ini page. It previously stated that scientists can die if you "overwork" them.
During my tests, my scientists kept dying even with the minimum possible research funding of 1k per day, so I came to the conclusion that they can always die while research is running, and the amount of funding has no influence on how likely they are to die.

I didn't check about that mentioned 1 million credits mark, so I'm not really sure what was meant by that.

**Reply by mefistotelis** [reply]
Yes, the probability doesn't change based on amount of work done. It can happen, as long as some funding was consumed by the research and some progress was made in the research. This code is called twice per day, for weapons and mods:
```
        // Each day there is a chance of a scientist dying
        if (LbRandomAnyShort() % 1000 < daily_scientist_death_chance_permil)
        {
            scientists_died++;
            if (LbRandomAnyShort() & 1)
                scientists_died++;
            if (scientists_died > research.Scientists)
                scientists_died = research.Scientists;
            research.Scientists -= scientists_died;
        }
```
So it looks like you can lose up to 4 scientists per day.

**Reply by Moburma** [reply]
For some reason I thought it was only if you went beyond the optimum money > research time boost value. I have no idea why I thought that in retrospect, so this is exactly what I mean, change away!

**Comment by Moburma** [comment]
I wrote an update on the main wiki page and also the FAQs page to explain Windows portable installs, and also where saved games are stored.

**Comment by Moburma** [comment]
I added a troubleshooting entry for black menu screen likely being a gfx files problem

**Comment by Moburma** [comment]
I updated the install instructions as I realised it didn't mention anything about needing files from the levels and gfx repositories if updating by hand, so clarified.

**Comment by Moburma** [comment]
Updated instructions on where the saved games location is now in non-portable mode and what to do if you are updating from an older version (you need to copy your saved games overs)

**Comment by mefistotelis** [comment]
Updated troubleshooting:
https://github.com/swfans/syndwarsfx/wiki/FAQS-&-Troubleshooting#when-i-start-the-game-i-get-a-black-screen-in-the-menus-with-the-game-running-behind-it

Added saves location on Linux, how to find execution logs, and improved music debug to mind Linux.

**Comment by Moburma** [comment]
Clarified on the main page that you need to have actually installed the GOG version first to use it as an install source.

**Reply by mefistotelis** [reply]
You never know what will monkey do unless you sit and observe. You think it will pry a square block into round hole, but it will eat the block instead.

**Comment by Moburma** [comment]
Added a small update on the FAQ page explaining how joystick controls work.
